radio/src/gui/gui_common.cpp

```diff
@@ -1252,6 +1252,9 @@ uint8_t MENU_FIRST_LINE_EDIT(const uint8_t * horTab, uint8_t horTabMax)
 
 uint8_t MODULE_BIND_ROWS(int moduleIdx)
 {
+  if (isModuleELRS(moduleIdx) && crossfireModuleStatus[moduleIdx].major >= 3 && crossfireModuleStatus[moduleIdx].minor >= 4)
```

Comment:
Won't this fail on 4.0 - 4.4?, etc?
```suggestion
  if (isModuleELRS(moduleIdx) &&
      (crossfireModuleStatus[moduleIdx].major > 3 ||
       (crossfireModuleStatus[moduleIdx].major == 3 &&
        crossfireModuleStatus[moduleIdx].minor >= 4)))
```